dev/Expander/Expander.xaml

```diff
@@ -0,0 +1,202 @@
+            to the toggle button. For this logic, several things were made:
+            1. Expander::OnApplyTemplate - We set the toggle button's events source to the expander's. This will announce expander properties
+               (expand/collapse) changes to narrator. Without this, narrator would announce the toggle button's "on/off" logic
+               instead of the "expand/collapse" logic.
+            2. ExpanderAutomationPeer::GetChildrenCore - Because of 1., we need to override GetChildrenCore to NOT include the toggle button.
+               If we don't do this, we create a weird dependency cycle between the toggle button and the expander.
+            3. ExpanderAutomationPeer::GetPeerFromPointCore Finally, when using narrator with a touch screen, this was overriden as well, to
+               focus the toggle button programmatically to sync the narrator focus with the keyboard focus.
+               Without this override, the narrator user that focuses the expander on a touch screen will see that pressing "Tab"
+               doesn't work how they would expect.-->
+        <Setter Property="IsTabStop" Value="False"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:Expander">
+                    <Grid>
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="ExpandStates">
+                                <VisualState x:Name="Expanded">
+                                    <VisualState.Storyboard>
+                                        <Storyboard>
+                                            <PopInThemeAnimation x:Name="PopIn" TargetName="ExpanderContent" FromVerticalOffset="-50" FromHorizontalOffset="0"/>
+                                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ExpanderContent" Storyboard.TargetProperty="Visibility">
+                                                <DiscreteObjectKeyFrame KeyTime="0" Value="Visible" />
+                                            </ObjectAnimationUsingKeyFrames>
+                                        </Storyboard>
+                                    </VisualState.Storyboard>
+                                </VisualState>
+                                <VisualState x:Name="Collapsed">
+                                    <VisualState.Storyboard>
+                                        <Storyboard>
+                                            <PopOutThemeAnimation TargetName="ExpanderContent" />
+                                        </Storyboard>
+                                    </VisualState.Storyboard>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="ExpandDirectionStates">
+                                <VisualState x:Name="Down" />
+                                <VisualState x:Name="Up">
```

Comment:
Does the same header style work for both down and up ? The chevron/transform would be different for up and down.